backend/models.py

```diff
@@ -36,13 +36,28 @@ class Participant(db.Model):
     email = Column(String(200))
     github = Column(String(200), default="")
     phone = Column(String(13))
+    hacknights = db.relationship("Hacknight",
+                                 secondary=participant_hacknight,
+                                 lazy='subquery',
+                                 backref=db.backref('participants', lazy=True))
 
```

Comment:
This looks like bidirectional many to many relationship. So either you should remove one relationship (from child class), or use back_populates instead of backref if you would like to keep this two directions.